src/main/java/me/arminb/sara/entities/BaseEntity.java

```diff
@@ -0,0 +1,33 @@
+package me.arminb.sara.entities;
+
+import java.util.Date;
+
+public class BaseEntity {
```

Comment:
Add two helper methods to this class:
1 - Modifies the modifiedAt field to the current date.
2 - Sets the createdAt field based on the object id